packages/repository-tests/src/crud/relations/fixtures/models/order.model.ts

```diff
@@ -38,6 +39,9 @@ export class Order extends Entity {
   @belongsTo(() => Customer)
   customerId: MixedIdType;
 
+  @belongsTo(() => Seller)
+  sellerId: number;
```

Comment:
```suggestion
  sellerId: MixedIdType;
```

We are using `string` when running on NoSQL databases like MongoDB and Cloudant.